doc/guides/bluetooth/autopts/autopts-win10.rst

```diff
@@ -0,0 +1,393 @@
+
+    wsl --shutdown
+
+After Ubuntu restart, go to:
+
+.. code-block::
+
+    cd /mnt/c/Users/codecoup/zephyrproject
+
+and test if west can build:
+
+.. code-block::
+
+    west build -p auto -b nrf52840dk_nrf52840 zephyr/tests/bluetooth/tester/
+
+From now on, you can build projects by typing in Windows's Git Bash:
+
+.. code-block::
+
+    wsl -d Ubuntu-20.04 -u codecoup -- bash -c -i "cd /mnt/c/Users/Codecoup/zephyrproject/ ; west build -p auto -b nrf52840dk_nrf52840 zephyr/tests/bluetooth/tester/"
+
+Install nrftools
+=================
+
+On Windows download latest nrftools (version >= 10.12.1) from site
+https://www.nordicsemi.com/Software-and-tools/Development-Tools/nRF-Command-Line-Tools/Download
+and run default install.
+
+.. image:: download_nrftools_windows.png
+   :height: 350
+   :width: 500
+   :align: center
+
+Connect devices
+================
+
+.. image:: devices_1.png
+   :height: 400
+   :width: 600
+   :align: center
+
+.. image:: devices_2.png
+   :height: 700
+   :width: 500
+   :align: center
+
+Flash board
+============
+
+In Device Manager find COM port of your nrf board. In my case it is COM3.
+
+.. image:: device_manager.png
+   :height: 400
+   :width: 450
+   :align: center
+
+In Git Bash, go to zephyrproject
+
+.. code-block::
+
+    cd ~/zephyrproject
+
+You can display flashing options with:
+
+.. code-block::
+
+    west flash --help
+
+and flash board with built earlier elf file:
+
+.. code-block::
+
+    west flash --skip-rebuild --board-dir /dev/ttyS2 --elf-file ~/zephyrproject/build/zephyr/zephyr.elf
+
+Note that west does not accept COMs, so use /dev/ttyS2 as the COM3 equivalent,
+/dev/ttyS2 as the COM3 equivalent, etc.(/dev/ttyS + decremented COM number).
+
+Setup auto-pts project
+=======================
+
+In Git Bash, clone project repo:
+
+.. code-block::
+
+    git clone https://github.com/codecoup/auto-pts.git
+
+Go into the project folder:
+
+.. code-block::
+
+    cd auto-pts
+
+Install required python modules:
+
+.. code-block::
+
+   pip3 install --user wheel
+   pip3 install --user -r autoptsserver_requirements.txt
+   pip3 install --user -r autoptsclient_requirements.txt
+
+Install socat.exe
+==================
+
+Download and extract socat.exe from https://sourceforge.net/projects/unix-utils/files/socat/1.7.3.2/
+into folder ~/socat-1.7.3.2-1-x86_64/.
+
+.. image:: download_socat.png
+   :height: 400
+   :width: 450
+   :align: center
+
+Add path to directory of socat.exe to PATH:
+
+.. image:: add_socat_to_path.png
+   :height: 400
+   :width: 450
+   :align: center
+
+Running AutoPTS
+================
+
+Server and client by default will run on localhost address. Run server:
+
+.. code-block::
+
+    python ./autoptsserver.py -S 65000
+
+.. image:: autoptsserver_run.png
+   :height: 200
+   :width: 800
+   :align: center
+
+Run client:
+
+.. code-block::
+
+    python ./autoptsclient-zephyr.py zephyr-master ~/zephyrproject/build/zephyr/zephyr.elf -t COM3 -b nrf52 -S 65000 -C 65001
```

Comment:
Ok, here I see another explicit nRF reference as well. Are these instructions meant to be applicable only to nRF boards? (that's not necessarily a problem, but it should then be made clear early on in the instructions)

Reply:
Yes, you're right :)